pkg/utils/utils_test.go

```diff
@@ -72,5 +72,19 @@ var _ = Describe("Utils", func() {
 				Expect(len(RandStringRunes(0))).To(Equal(0))
 			})
 		})
+
+		Context("Test RemoveDuplicateStrings", func() {
+			It("Should remove duplicate strings", func() {
+				input := []string{"1", "1", "2", "3", "4", "3", "5", "1"}
```

Comment:
empty input list

Reply:
used DescribeTable to test both empty slices and nil slices are returned unchanged